programs/cleos/main.cpp

```diff
@@ -3204,14 +3207,13 @@ int main( int argc, char** argv ) {
    wrap->require_subcommand();
 
    // wrap exec
-   string wrap_con = "eosio.wrap";
    executer = "";
    string trx_to_exec;
    auto wrap_exec = wrap->add_subcommand("exec", localized("Execute a transaction while bypassing authorization checks"));
    add_standard_transaction_options(wrap_exec);
    wrap_exec->add_option("executer", executer, localized("Account executing the transaction and paying for the deferred transaction RAM"))->required();
    wrap_exec->add_option("transaction", trx_to_exec, localized("The JSON string or filename defining the transaction to execute"))->required();
-   wrap_exec->add_option("--contract,-c", wrap_con, localized("The account which controls the wrap contract"));
+   wrap_exec->add_option("--contract,-c", wrap_cntr_name, localized("The account which controls the wrap contract"));
```

Comment:
here is the same as with `con`, return `wrap_con` variable and just change it's default value to const. actually, cleos will not build now, because this call can't accept constant